auto_tests/toxav_mt_test.cc

```diff
@@ -0,0 +1,498 @@
+#include <algorithm>
+#include <atomic>
+#include <chrono>
+#include <cstdio>
+#include <cstring>
+#include <iostream>
+#include <memory>
+#include <mutex>
+#include <thread>
+#include <vector>
+
+#include "../testing/misc_tools.h"
+#include "../toxav/toxav.h"
+#include "../toxcore/tox.h"
+#include "check_compat.h"
+
+// Make lines shorter
+using Clock = std::chrono::high_resolution_clock;
+using TimePoint = std::chrono::time_point<Clock>;
```

Comment:
The naming convention in this file is a bit all over the place. E.g. `TimePoint` and `Tox_Ptr`. Also, some top level functions are `snake_case`, others are `dromedaryCase`. Since you're not using the Tox naming conventions (which are somewhat self-described by all the other sources), can you briefly describe them at the top of the file? Just a list of "kind of identifier: naming convention".

Reply:
Should be mirroring random_testing.cc now